dev/release/README.md

```diff
@@ -169,6 +169,15 @@ To generate the changelog, set the `GITHUB_TOKEN` environment variable and then
 providing two commit ids or tags followed by the version number of the release being created. For example,
 to generate a change log of all changes between the `50.3.0` tag and `branch-51`, in preparation for release `51.0.0`:
 
+> [!NOTE]
```

Comment:
You can see a rendered preview here: https://github.com/alamb/datafusion/blob/alamb/more_release_update/dev/release/README.md#changelog-generation

<img width="1071" height="212" alt="Image" src="https://github.com/user-attachments/assets/c7989d09-7cf7-463a-a71c-8770140a77dd" />